.travis_install.bash

```diff
@@ -81,6 +86,24 @@ case "${TRAVIS_OS_NAME}:${LLVM_CONFIG}" in
     #export PATH=llvmsym/:$PATH
   ;;
 
+  "osx:llvm-config-4.0")
+    brew update
+    brew install shellcheck
+    shellcheck ./.*.bash ./*.bash
+
+    brew install pcre2
+    brew install libressl
+
+    brew install llvm@4.0
+    brew link --overwrite --force llvm@4.0
+    mkdir llvmsym
+    ln -s "$(which llvm-config)" llvmsym/llvm-config-4.0
+    ln -s "$(which clang++)" llvmsym/clang++-4.0
```

Comment:
im not sure that these are needed. i'd say turn these 3 lines and the corresponding export PATH off and we can enable if needed.